test/benchmark/benchmark_alphanum_test.go

```diff
@@ -0,0 +1,75 @@
+package benchmark
+
+import (
+	"regexp"
+	"testing"
+
+	validation "github.com/go-ozzo/ozzo-validation/v4"
+	"github.com/go-playground/validator/v10"
+	"github.com/gookit/validate"
+
+	"github.com/sivchari/govalid/test"
+)
+
+func BenchmarkGoValidAlphanum(b *testing.B) {
+	instance := test.Alphanum{ProductCode: "product2024", SerialNumber: "ABC123", Username: "username"}
+	b.ResetTimer()
+	for b.Loop() {
+		err := test.ValidateAlphanum(&instance)
+		if err != nil {
+			b.Fatal("unexpected error:", err)
+		}
+	}
+	b.StopTimer()
+}
```

Comment:
```suggestion
func BenchmarkGoValidAlphanum(b *testing.B) {
	instance := test.Alphanum{ProductCode: "product2024", SerialNumber: "ABC123", Username: "username"}
	for b.Loop() {
		err := test.ValidateAlphanum(&instance)
		if err != nil {
			b.Fatal("unexpected error:", err)
		}
	}
}
```

b.ResetTimer and b.StopTimer isn't necessary anymore. So please remove it.